src/packages/searchbar/searchbar.tsx

```
@@ -48,147 +57,215 @@
+      setTimeout(() => {
+        setInnerTag(event.target?.value ? tag : false)
+      }, 150)
+    },
+    [onBlur, tag, value]
+  )
+  const clearaVal = useCallback(
+    (event: MouseEvent<HTMLDivElement>) => {
+      if (disabled || readOnly) return
+      setValue('')
+      forceFocus()
+      onChange && onChange('')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)